shopping-cart/.dockerignore

```diff
@@ -0,0 +1,4 @@
+Dockerfile
+.dockerignore
```

Comment:
Curious what is the reason to put `Dockerfile` and `.dockerignore` here?

Reply:
I've added that because otherwise those files are copied by `COPY . .`. However, that is not a big deal since those files are small, and they are not copied to the second stage (production image) anyway. I'll remove them.